managing_webhooks.md

```diff
@@ -0,0 +1,47 @@
+# Managing Webhooks
+
+## Webhook Manager Role
+
+Viewing, creating, and editing webhooks is relegated to accounts that have the Webhooks Manager Role. When logged in with an account with the Webhooks Manager Role, there is a navigation button named Webhooks in the navigation bar on the left. Clicking this button will take you to the Webhooks Management page where new webhooks can be created or existing webhook can be viewed or edited. To have the Webhooks Manager Role added to an account, please contact ActiGraph support.
```

Comment:
" webhooks is relegated to accounts that have the Webhooks Manager Role"  maybe something here to say this is a CentrePoint permission to clarify "accounts"